### ISSUE TITLE
Error installing on MacOS Catalina

### ISSUE THREAD

**Issue by eugenio11** [microsoft/msphpsql#1250]
I have tried to install on MacOS Catalina, php 7.3.24 on a MAMP installation

sudo /Applications/MAMP/bin/php/php7.3.24/bin/pecl install sqlsrv pdo_sqlsrv

but I got:

/private/tmp/pear/install/sqlsrv/php_sqlsrv.h:25:10: fatal error: 'php.h' file not found

any suggestions?

Thanks!

**Comment by yitam** [comment]
Have you also contacted [MAMP](https://documentation.mamp.info/en/MAMP-Mac/) support?
It seems that `php.h` is not in the include path, please run [php-config](https://www.php.net/manual/en/install.pecl.php-config.php)

Also, do you know php 7.3 is near [the end of support](https://www.php.net/supported-versions.php)?

**Comment by eugenio11** [comment]
Thanks for your reply. PHP 7.3 was just a test, I had the same issue with PHP 7.4.

I'll write here what I did and how I solved this, in case someone could find this useful.

1) I have tried to install PHP through macports, then I realized that macports doesn't provide a pdo_sqlsrv port so I uninstall it

2) I've tried again to install sqlsrv and pdo_sqlsrv via pecl and this time I got a different error (don't know if this was related to the macports installation or something different that I did):
error: Cannot find php_pdo_driver.h.

So I realized that the bin_dir in my pecl configuration was set to the wrong PHP directory. I think this was due to the fact that on that machine many installation tests have been done with multiple PHP versions (the one provided by macOS, the ones provided by mamp, the one provided by macports)

3) I set the correct bin_dir
pecl config-set bin_dir /Applications/MAMP/bin/php/php7.4.12/bin/

and everything worked.
